packages/kbn-es/src/utils/native_realm.js

```diff
@@ -13,15 +13,12 @@ const { log: defaultLog } = require('./log');
 
 exports.NativeRealm = class NativeRealm {
   constructor({ elasticPassword, port, log = defaultLog, ssl = false, caCert }) {
-    this._client = new Client({
-      node: `${ssl ? 'https' : 'http'}://elastic:${elasticPassword}@localhost:${port}`,
-      ssl: ssl
-        ? {
-            ca: caCert,
-            rejectUnauthorized: true,
-          }
-        : undefined,
-    });
+    const auth = { username: 'elastic', password: elasticPassword };
```

Comment:
__note:__ this change wasn't required, but it feels like a more correct way to use `elasticsearch` client with the predefined credentials.